model/quic-socket-tx-pfifo-scheduler.h

```diff
@@ -0,0 +1,96 @@
+/* -*- Mode:C++; c-file-style:"gnu"; indent-tabs-mode:nil; -*- */
+/*
+ * Copyright (c) 2019 SIGNET Lab, Department of Information Engineering, University of Padova
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation;
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ * Authors: Alvise De Biasio <alvise.debiasio@gmail.com>
+ *          Federico Chiariotti <chiariotti.federico@gmail.com>
+ *          Michele Polese <michele.polese@gmail.com>
+ *          Davide Marcato <davidemarcato@outlook.com>
+ *          
+ */
+
+#ifndef QUICSOCKETTXPFIFOSCHEDULER_H
+#define QUICSOCKETTXPFIFOSCHEDULER_H
+
+#include "quic-socket-tx-scheduler.h"
+#include <queue>
+#include <vector>
+
+namespace ns3
+{
+
+/**
+ * \brief The PFIFO implementation
+ *
+ * This class is a Priority FIFO implementation of the socket scheduler, which prioritizes streams with a lower stream number
+ */
+class QuicSocketTxPFifoScheduler: public QuicSocketTxScheduler
+{
+public:
+	/**
+	 * \brief Get the type ID.
+	 * \return the object TypeId
+	 */
+	static TypeId GetTypeId(void);
+
+	QuicSocketTxPFifoScheduler();
+	QuicSocketTxPFifoScheduler(const QuicSocketTxPFifoScheduler &other);
+	~QuicSocketTxPFifoScheduler(void);
+
+	/**
+	 * Print the scheduler buffer information to a string
+	 *
+	 * \param os the std::ostream object
+	 */
+	void Print(std::ostream &os) const;
```

Comment:
Add override at the end of the function declarations for the functions that you override